go.mod

```diff
@@ -17,7 +17,7 @@ require (
 )
 
 replace (
-	github.com/gardener/machine-controller-manager => github.com/rkjakeer/machine-controller-manager v0.38.1-0.20210701092429-b810a051db73
+	github.com/gardener/machine-controller-manager => github.com/rkjakeer/machine-controller-manager v0.38.1-0.20210708085258-8fd87c692191
```

Comment:
Leaving a comment, so that we remember to update the go.mod file with proper vendoring.